### ISSUE TITLE
Releasing duplicate Pokemons doesn't work in new-api-update branch

### ISSUE THREAD

**Issue by bertjan** [BoBeR182/PoGoBoWo#70]
## Expected Behavior
Duplicate pokemons are released.

## Actual Behavior
Duplicate pokemons are only partly released (a lot of duplicates are not released). For example:
`Rattata CP 400; IV 77.78; ST 47/47; IS 12; ID 14; IS 9;
Rattata CP 274; IV 75.56; ST 38/38; IS 11; ID 15; IS 8;
Rattata CP 232; IV 82.22; ST 37/37; IS 10; ID 15; IS 12;
Rattata CP 147; IV 64.44; ST 28/28; IS 10; ID 12; IS 7;
Rattata CP 131; IV 62.22; ST 26/26; IS 10; ID 11; IS 7;
Rattata CP 118; IV 26.67; ST 24/24; IS 10; ID ; IS 2;
Rattata CP 116; IV 68.89; ST 25/25; IS 8; ID 14; IS 9;
Rattata CP 110; IV 13.33; ST 25/25; IS 1; ID 1; IS 4;
Rattata CP 99; IV 55.56; ST 24/24; IS 12; ID 4; IS 9;
Rattata CP 92; IV 37.78; ST 23/23; IS 6; ID 5; IS 6;
Rattata CP 91; IV 37.78; ST 21/21; IS 6; ID 10; IS 1;
Rattata CP 88; IV 73.33; ST 23/23; IS 10; ID 10; IS 13;
Rattata CP 82; IV 51.11; ST 20/20; IS 12; ID 8; IS 3;
Rattata CP 81; IV 46.67; ST 21/21; IS 11; ID 4; IS 6;
Rattata CP 78; IV 37.78; ST 19/19; IS 10; ID 7; IS ;
Rattata CP 68; IV 57.78; ST 20/20; IS 9; ID 7; IS 10;
Rattata CP 49; IV 31.11; ST 15/15; IS 9; ID 3; IS 2;
Rattata CP 39; IV 64.44; ST 15/15; IS 11; ID 6; IS 12;
Rattata CP 36; IV 40; ST 14/14; IS 12; ID ; IS 6;
Rattata CP 23; IV 68.89; ST 12/12; IS 10; ID 6; IS 15;
Rattata CP 10; IV 44.44; ST 10/10; IS 11; ID ; IS 9;
Rattata CP 10; IV 55.56; ST 10/10; IS 11; ID 14; IS ;`

### Environmental Information

Operating System: Linux
Commit:  63bd005
config.json:
relevant piece:
`        "MIN_KEEP_IV" : 65,
        "KEEP_CP_OVER": 1000,
        "RELEASE_DUPLICATE": true,
        "DUPLICATE_CP_FOREGIVENESS": 0.90,
        "STEP_SIZE": 100,
        "RETURN_START_INTERVAL": 20,
        "MAX_BALL_TYPE": 2,
        "SLOW_BUT_STEALTH": false,
        "AUTO_HATCHING": true,`

## Steps to reproduce

1. Switch to new-api-update branch
2. Set config as above
3. wait a bit

**Comment by bertjan** [comment]
@etrevis the same issue exists in your fork (which isn't very unexpected since both repo's are pretty much up-to-date to each other)

**Comment by etrevis** [comment]
@bertjan I also noticed it, and there are also problems if `"RELEASE_DUPLICATE": false` and evolutions, working on it ;9

**Comment by BoBeR182** [comment]
@etrevis maybe revert to some of the code in https://github.com/BoBeR182/PoGoBoWo/blob/4eff41b2f66026db4920a3dc6fd8358fa83c961e/pgoapi/pgoapi.py#L384 As we know our evolution and release logic worked, outside of it trying to release already evolved pokemon (noncrashing bug) and releasing pokemon in gyms (also just a logging display error).

**Comment by etrevis** [comment]
@BoBeR182 yes maybe reverting is better for the release part. The evolution part is unchanged, i've only updated indices to reflect new api responses
